packages/docusaurus-plugin-content-blog/src/__tests__/authors.test.ts

```diff
@@ -227,7 +227,7 @@ describe('getBlogPostAuthors', () => {
       {
```

Comment:
to make this test more realistic/consistent with real world usage, I'd use `imageURL: '/baseUrl/img/slorber.png',` for `slorber` because the authors map is supposed to be provided with the baseUrl already